### PR TITLE
[4.2] Fixing labels for Smart Search menu item

### PR DESCRIPTION
**Pull request by Hackwar**
The language strings used for the 2 request parameter options in the search view of com_finder have translation keys which have a wrong english translation and which are used in 2 different circumstances for different stuff. This adds additional translation keys and "right" translations.

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on f28c99eabb3af08f9cb5b2ba737256ab97750d0b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38228">issues.joomla.org/tracker/joomla-cms/38228</a>.</sub>

**Comment by brianteeman** [comment]
please alpha sort language strings

and why add a new string with identical text<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38228">issues.joomla.org/tracker/joomla-cms/38228</a>.</sub>

**Comment by Hackwar** [comment]
Because they are used in pretty different situations.

File: administrator/language/en-GB/com_finder.ini [hunk 1]
```diff
@@ -218,6 +219,7 @@ COM_FINDER_SEARCH_LABEL="Search %s:"
 COM_FINDER_SEARCH_SEARCH_QUERY_DESC="Search in content map title."
 COM_FINDER_SEARCH_SEARCH_QUERY_LABEL="Search Content Maps"
 COM_FINDER_SEARCHES_TABLE_CAPTION="Table of Search Statistics"
+COM_FINDER_SELECT_FILTER_LABEL="Select filter"
```
Comment (Quy) [inline administrator/language/en-GB/com_finder.ini]:
```suggestion
COM_FINDER_SELECT_FILTER_LABEL="Filter"
```

File: administrator/language/en-GB/com_finder.ini [hunk 2]
```diff
@@ -206,6 +206,7 @@ COM_FINDER_NO_ERROR_RETURNED="No error was returned. Make sure error reporting i
 COM_FINDER_NO_FILTERS="No filters have been created yet."
 COM_FINDER_NO_RESULTS="No results match your search criteria."
 COM_FINDER_NO_RESULTS_OR_FILTERS="No results match your search criteria or no filters have been created yet."
+COM_FINDER_PREFILL_SEARCH_LABEL="Pre-fill search"
```
Comment (Quy) [inline administrator/language/en-GB/com_finder.ini]:
```suggestion
COM_FINDER_PREFILL_SEARCH_LABEL="Prefilled Search Term"
```

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 7225f8dd5c3062d4e53b06a1535fce47f228dd9e<br /><br />Thanks, this label is much better than before.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38228">issues.joomla.org/tracker/joomla-cms/38228</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 7225f8dd5c3062d4e53b06a1535fce47f228dd9e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38228">issues.joomla.org/tracker/joomla-cms/38228</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38228">issues.joomla.org/tracker/joomla-cms/38228</a>.</sub>

**Comment by roland-d** [comment]
Thanks everybody

**Pull request merged**